frontend/src/components/processMeasurements/ProcessMeasurementsDetailContent.js

```diff
@@ -76,15 +76,11 @@ const ProcessMeasurementsDetailContent = ({
               <TrackingFieldsContent entity={processMeasurement}/>
             </TabPane>
             <TabPane tab="Properties" key="2" style={{marginTop:8} }>
-              { processMeasurement?.properties?.length > 0 &&
-                  <>
-                  <Title level={3} style={{marginTop: '20px'}}>Properties</Title>
-                    <ProcessProperties
-                        propertyIDs={processMeasurement.properties}
-                        protocolName={protocolsByID[processMeasurement.protocol]?.name}
-                    />
-                  </>
-               }
+              <Title level={3} style={{marginTop: '20px'}}>Properties</Title>
+              <ProcessProperties
```

Comment:
It would be useful to display the Process properties in the Process Measurement Detail Content page as well. Like that we have both: Process' properties and process measurement's properties in both pages (process measurement detail content and process detail content).